### ISSUE TITLE
NPM package?

### ISSUE THREAD

**Issue by nybblr** [seethroughtrees/casperjs-spider#10]
Any interest in publishing this as a CLI tool on NPM?

I have have a Travis CI build running on a static site that checks for various errors, and I'd like to be able to run the crawler during the build to make sure links/JavaScript isn't getting broken on the branch.

To do that, Travis would need a package it could grab and install.

**Comment by seethroughtrees** [comment]
Definitely.  I'm working on a 2.0 that will have its own bin and be available on npm.  If not being on npm is a blocker for you, i can push up the current version.  Otherwise, I'd just as soon wait until the new one is done.

**Comment by nybblr** [comment]
Awesome, what timeframe are you thinking?

If you think it will be >2 weeks before the 2.0 release, a release for the current version would be great if it isn't any trouble.

Thanks!

**Comment by seethroughtrees** [comment]
Yep.  Let me see if I can get it done this week.  If I don't have a 2.0 by then, i'll throw the old one up there.

**Comment by nybblr** [comment]
:+1:

**Comment by PierreAmmeloot** [comment]
Hello, do you have any news about NPM Package?
I think it's the best idea in order to promote this project on Open Source community. :-)
